Core/Filesystem/test/nandfs_tests.c

```diff
@@ -79,16 +79,24 @@ int check_all_pages_in_block(int block, char c) {
     }
 }
 
-int write_all_pages_in_block(int block, char c) {
+int read_from_block_one(uint16_t page) {
```

Comment:
Why did you delete the other test?

Reply:
Hmm, do not know why I did that, nevertheless I have added it back